lib/active_support/cache/memcached_store.rb

```diff
@@ -241,9 +241,8 @@ def cas_raw?(options)
 
       def expiration(options)
         expires_in = options[:expires_in].to_i
-        if expires_in > 0 && !options[:raw]
-          # Set the memcache expire a few minutes in the future to support race condition ttls on read
-          expires_in += 5.minutes.to_i
+        if expires_in > 0 && options[:race_condition_ttl] && !options[:raw]
```

Comment:
The `:raw` option is used to support using the memcached increment and decrement operations.  The condition is still appropriate since it doesn't make sense to extend the expiration when using `raw: true` because it only works when serializing the entry into the cache.  An alternative would be to raise if both `raw: true` and `race_condition_ttl` options are provided.